vcpkg-artifacts/yaml/yaml-types.ts

```diff
@@ -175,10 +175,13 @@ export /** @internal */ abstract class Yaml<ThisType extends Node = Node> {
 
   /** if this node has any data, this should return false */
   get empty(): boolean {
-    if (isCollection(this.node)) {
-      return !(this.node?.items.length);
-    }
-    return !isNullish(this.node?.value);
+      if (isCollection(this.node)) {
+          return !(this.node?.items.length);
+      } else if (isScalar(this.node)) {
+          return !isNullish(this.node.value);
+      }
+
+      return false;
```

Comment:
Should this be an assertion? What node isn't a collection or scalar?

Reply:
No idea. I'm just trying to mimic what the old behavior was doing. Previously if it wasn't a scalar then `this.node?.value` would have been `null`, so `isNullish` would return `true`, so the result was `false`. (The previous version of TypeScript accepted this, the new one does not)